CONTRIBUTING.md

```diff
@@ -27,7 +27,7 @@ land your contribution below.
    bugs, and whether the contribution aligns with what was previously discussed.
    If you think that a review comment misses something or is not quite right,
    please challenge it!
-6. If the review passes, your PR will be merged and ship in the next version of
+6. If the review passes, your PR will be merged and shipped in the next version of
```

Comment:
I think the previous phrasing was fine? 
- your PR will be merged
- your PR will ship in the next version
- -> your PR will be merged and ship in the next version